test/integration/routes_test.rb

```diff
@@ -112,5 +112,9 @@ class RoutesTest < ActionDispatch::IntegrationTest
   test "test get request for sets calibrated" do
     assert_routing({ path: '/sets/calibrated', method: 'get' }, {controller: 'sets', action: 'show', id: 'calibrated'})
   end
+	
+  test "test get request for spectrums anonymous" do
+    assert_routing({ path: '/spectrums/anonymous', method: 'get' }, {controller: 'spectrums', action: 'anonymous'})
```

Comment:
@cmcgee0308 You can fix it by updating this line to this
```suggestion
    assert_routing({ path: '/spectrums/anonymous', method: 'get' }, {controller: 'spectrums', action: 'show', id: 'anonymous'})
```

Reply:
Okay I made the changes!